modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/rel/TableScanNode.java

```diff
@@ -218,7 +224,7 @@ private void requestNextBatch() {
     }
 
     private class SubscriberImpl implements Flow.Subscriber<BinaryRow> {
-        private int received;
+        int received = 0; // HB defenced here.
```

Comment:
why do you remove the access modifier?

Reply:
just forgot to push last changes, sorry (